packages/config-plugins/src/utils/Updates.ts

```diff
@@ -1,10 +1,16 @@
-import { ExpoConfig } from '@expo/config-types';
+import { Android, ExpoConfig, IOS } from '@expo/config-types';
+import { getRuntimeVersionForSDKVersion } from '@expo/sdk-runtime-versions';
```

Comment:
@esamelson did you have a reason for wanting to put `getRuntimeVersion` into a fully separate module like `getRuntimeVersionForSDKVersion` instead of just putting it in `config-plugins` like this PR is proposing?

Reply:
No, I meant "module" as in "ts file in a package" rather than "package," but I suppose I could have just written "method" instead. Sorry for the confusion! I think this is good, as long as this method is also accessible by the publish scripts (which it looks like it is / will be).